pkgs/applications/graphics/oculante/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, pkg-config
+, openssl
+, fontconfig
+, nasm
+, netcat
+, libX11
+, libXcursor
+, libXrandr
+, libXi
+, libGL
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "oculante";
+  version = "0.6.34";
+
+  src = fetchFromGitHub {
+    owner = "woelper";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Nt6JOEtFkIgHojw3+YO+g79/9rwk8C1OV9SR+2oXYEg=";
+  };
+
+  cargoSha256 = "sha256-XuVRNX3+E51dklL0+Er1PvL2THqzUwtZZHw8VlMyZAs=";
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+  ];
+
+  checkFlagsArray = [ "--skip=tests::net" ]; # requires network access
+
+  buildInputs = [
+    openssl
+    fontconfig
+    nasm
```

Comment:
this should probably go in nativeBuildInputs

Reply:
ummm, tried but does not work

Reply:
I tried moving it to nativeBuildInputs and it just worked